src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styles from './Pagination.module.sass'
+import { useDispatch, useSelector } from 'react-redux'
+import { setFilter } from '../../store/slices/ProductSlice'
+
+function Pagination() {
+
+    const dispatch = useDispatch()
+
+    const pagination = useSelector( state => state.products.pagination)
+    const filter = useSelector( state => state.products.filter)
```

Comment:
```suggestion
    const { pagination, filter } = useSelector({ products } => products)
```
Nhưng đống này phải được truyền dưới dạng props chứ nhỉ? Vì pagination có thể cho nhiều thứ khác mà, như blog, review (giả sử trong trường hợp Website ngoài bán hàng ra còn có 1 trang đánh giá, review sản phẩm) chứ k chỉ riêng products